kotlinx-coroutines-core/jvm/test/linearizability/LockFreeTaskQueueLCStressTest.kt

```diff
@@ -5,89 +5,48 @@
 
 package kotlinx.coroutines.linearizability
 
-import com.devexperts.dxlab.lincheck.*
-import com.devexperts.dxlab.lincheck.annotations.*
-import com.devexperts.dxlab.lincheck.paramgen.*
-import com.devexperts.dxlab.lincheck.strategy.stress.*
 import kotlinx.coroutines.*
 import kotlinx.coroutines.internal.*
+import org.jetbrains.kotlinx.lincheck.annotations.*
+import org.jetbrains.kotlinx.lincheck.annotations.Operation
+import org.jetbrains.kotlinx.lincheck.paramgen.*
+import org.jetbrains.kotlinx.lincheck.verifier.*
+import org.jetbrains.kotlinx.lincheck.verifier.quiescent.*
 import kotlin.test.*
 
-internal data class Snapshot(val elements: List<Int>, val isClosed: Boolean) {
-    constructor(q: LockFreeTaskQueue<Int>) : this(q.map { it }, q.isClosed())
-}
-
-@OpGroupConfig.OpGroupConfigs(OpGroupConfig(name = "consumer", nonParallel = true))
-@Param(name = "value", gen = IntGen::class, conf = "1:3")
-class SCLockFreeTaskQueueLCStressTest : LockFreeTaskQueueLCTestBase() {
-    private val q: LockFreeTaskQueue<Int> = LockFreeTaskQueue(singleConsumer = true)
+@Param(name = "value", gen = IntGen::class, conf = "1:5")
+internal abstract class LockFreeTaskQueueWithoutRemoveLCStressTest(singleConsumer: Boolean) : VerifierState() {
+    @JvmField
+    protected val q = LockFreeTaskQueue<Int>(singleConsumer = singleConsumer)
 
     @Operation
     fun close() = q.close()
 
     @Operation
     fun addLast(@Param(name = "value") value: Int) = q.addLast(value)
 
-    /**
-     * Note that removeFirstOrNull is not linearizable w.r.t. to addLast, so here
-     * we test only linearizability of close.
-     */
-//    @Operation(group = "consumer")
-//    fun removeFirstOrNull() = q.removeFirstOrNull()
```

Comment:
Please, leave this commented out section together with comment in the code.

Reply:
We have tests with `removeFirstOrNull`